README.adoc

```diff
@@ -51,7 +51,49 @@ You can **run the https://github.com/rust-lang/rust-clippy[Clippy linter]** usin
 
 == Working in Omicron
 
-Omicron is a pretty large repo containing a bunch of related components.  The repo is structured as one large Cargo workspace.  When you run `cargo build` or `cargo test`, you'll wind up building and testing everything.  In practice, people are usually only working on one or two components at a time.  You can usually iterate faster by specifying a specific package, like `cargo build -p omicron-nexus`.
+Omicron is a pretty large repo containing a bunch of related components.  If you just build the whole thing with `cargo build` or `cargo test`, it can take a while, even for incremental builds.  Since most people are only working on a few of these components at a time, it's helpful to be know about Cargo's tools for working with individual packages in a workspace.
+
+NOTE: This section assumes you're already familiar with the prerequisites and environment setup needed to do _any_ work on Omicron.  See xref:docs/how-to-run-simulated.adoc[] or xref:docs/how-to-run.adoc[] for more on that.
```

Comment:
I think it might be good to call out there are documents on running omicron, but this section mostly focuses on iterative development, so I'm not sure it's really a prerequisite?

Reply:
I wanted to mention this because you have to run the `install_builder_prerequisites.sh` script and set up your environment for any of this to work.  As a new user I hate the feeling when the instructions don't work and I know I've missed some steps the author just assumed but I can't figure out what they were.